client/cli/cli.go

```diff
@@ -18,6 +20,7 @@ import (
 	"github.com/peterh/liner"
 )
 
+const configDir = ".dlv"
```

Comment:
Probably want to think about win32 vs posix where you probably want different locations.